src/shared/dumb_components/member_fields.js

```diff
@@ -19,12 +19,18 @@ const PersonalFields = (
   , inputClassName
   , memberView
   , active_tab: member_fields
+  , update_membership_type
   }
 ) => {
-  const fs = ((fields.membership_type && fields.membership_type.value) || '').match('life')
+  let fs = ((fields.membership_type && fields.membership_type.value) || '').match('life')
```

Comment:
If we went with this, it would make sense to still have `const` and just have a pipeline of functions. Going to take another look at this approach though